README.md

```diff
@@ -95,11 +95,13 @@ Wechaty already held lots of talk and got a lot of blogs in the past 4 years, he
 
 import { WechatyBuilder } from 'wechaty'
 
-WechatyBuilder.build()  // get a Wechaty instance
-  .on('scan', (qrcode, status) => console.log(`Scan QR Code to login: ${status}\nhttps://wechaty.js.org/qrcode/${encodeURIComponent(qrcode)}`))
-  .on('login',            user => console.log(`User ${user} logged in`))
-  .on('message',       message => console.log(`Message: ${message}`))
-  .start()
+const wechaty = WechatyBuilder.build({}) // get a Wechaty instance
+
+wechaty.on('scan', (qrcode, status) => console.log(`Scan QR Code to login: ${status}\nhttps://wechaty.js.org/qrcode/${encodeURIComponent(qrcode)}`))
+  .on('login', (user) => console.log(`User ${user} logged in`))
```

Comment:
Please keep the spaces so that the `=>` can be aligned across lines like the original style.